sklearn/tree/_export.py

```diff
@@ -668,7 +668,11 @@ def export(self, decision_tree, ax=None):
             # get figure to data transform
             # adjust fontsize to avoid overlap
             # get max box width and height
-            extents = [ann.get_bbox_patch().get_window_extent() for ann in anns]
+            extents = [
+                ann.get_bbox_patch().get_window_extent()
+                for ann in anns
+                if ann.get_bbox_patch() is not None
+            ]
```

Comment:
```suggestion
            extents = [
                bbox_patch.get_window_extent()
                for ann in anns
                if (bbox_patch := ann.get_bbox_patch()) is not None
            ]
```

This avoids duplicate calls to `ann.get_bbox_patch()` though it probably is not a big deal.